MonoGame.Framework/Graphics/Viewport.cs

```diff
@@ -58,6 +21,10 @@ public struct Viewport
 		private float maxDepth;
 		
 		#region Properties
+
+        /// <summary>
+        /// The height of this <see cref="Viewport"/>.

```

Comment:
We don't need to include `of this <see cref="Viewport"/>` on every summary.  Also the link is redundant as it is just linking to the same docs they are already reading.